docs/iris/src/whatsnew/latest.rst.template

```diff
@@ -10,37 +10,50 @@ This document explains the changes made to Iris for this release
    :depth: 3
 
 
-Features
-========
+🎉 Announcements
```

Comment:
```suggestion
📢 Announcements
```

Reply:
Yup, love it. I'm sold on that too :smile: